pkg/util/certificates.go

```diff
@@ -50,6 +50,7 @@ func GetClientTLSConfig(caFile, certFile, keyFile, serverName string) (*tls.Conf
 		tlsConfig := &tls.Config{
 			RootCAs:    certPool,
 			MinVersion: tls.VersionTLS12,
+			NextProtos: protos,
```

Comment:
Can we rewire this to only set the field with protos if protos was set as an input? I'd rather not modify the tlsConfig default value if the user did not set the --alpn-proto parameter.

Reply:
I tested to make sure that the defaults are still used given a zero length NextProtos, but you're right — better to be safe here. Fixed.